File: Modul6/C/PRAK602-2310817210003-MuhammadAzrianzan.c
#include <stdio.h>

int main() {
    int ruangan;
    scanf("%d", &ruangan);
    int array[ruangan];
    int array2[ruangan];
    for (int i = 0; i < ruangan; i++) {
        scanf("%d", &array[i]);
    }

    for (int j = 0; j < ruangan; j++) {
        int zetsu = array[j] * (j + 1);
        array2[j] = zetsu;
    }
    
    for (int x = 0; x < ruangan; x++) {
        printf("%d ", array2[x]);
    }
    return 0;
}

File: Modul6/Python/PRAK602-2310817210003-MuhammadAzrianzan.py
ruangan = int(input(""))
list1 = list(map(int, input().split()))
list2 = []

for i in range(ruangan):
    zetsu = list1[i] * (i + 1)
    list2.append(zetsu)

for j in range(ruangan):
    print(list2[j], end=" ")